webviz_ert/controllers/multi_response_controller.py

```diff
@@ -196,4 +202,32 @@ def _generate_plot(ensemble_id: str, color: str) -> Optional[ResponsePlotModel]:
             for observation in plot._observations:
                 fig.add_trace(observation.repr)
         fig.update_layout(assets.ERTSTYLE["figure"]["layout"])
+        fig.update_layout({"xaxis": {"title": {"text": x_axis_label}}})
+        fig.update_layout(assets.ERTSTYLE["figure"]["layout-value-y-axis-label"])
         return fig
+
+
+def axis_label_for_ensemble_response(
+    ensemble: "EnsembleModel", responseName: str
+) -> str:
+    response = ensemble.responses[responseName]
+    response_data = response.data
+    index = response_data.index
+    index_type = type(index)
+    data_sample = index[0]
+    if index_type == pd.core.indexes.base.Index:
+        # try to convert to date-time
```

Comment:
this comment is not necessary as i think the code explains itself :+1:

Reply:
indeed : ) fixed!